static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -22,11 +22,23 @@ export default class FilterVisibilityToggle extends SearchkitComponent {
     return checkFilterVisibility(filterName) ? "" : "closed";
   };
 
-  isInResults: Function = (id: string): boolean => {
-    if (this.getResults()) {
-      const elmId = FILTER_ID_ADJUST[id] || id;
-      const docCount = _.get(this.getResults(), ['aggregations', elmId, 'doc_count'], 0);
+  getChildFacetDocCount: Function = (results: Object, resultId: string): number => {
+    const elementResult = _.get(results, ['aggregations', resultId]);
+    if (elementResult === undefined) {
+      return 0;
```

Comment:
good catch. this logic was rendered obsolete by [this line](https://github.com/mitodl/micromasters/pull/2082/files#diff-50babb8b383f61e9381381b263c51697R36). i got rid of these 2 lines